docs/it-ams-monitoring-data.md

```diff
@@ -0,0 +1,53 @@
+# IT-AMS Monitoring Data Integration Guide
+
+## Overview
+The TTA Hub integrates with IT-AMS to retrieve and process monitoring data related to Head Start recipients’ reviews and findings. The main steps are:
+- Monitoring review data is securely transferred from IT-AMS.
+- The system automatically imports the data and stores it in the database (PostgreSQL).
+- The findings generate automatic goals for grant recipients to address compliance issues.
+- The HSES also receives this data for program management visibility. This happens in parallel to the TTA Hub's import.
+
+## How the Integration Works
+1. IT-AMS generates XML data files containing monitoring reviews daily.
+2. The TTA Hub downloads the data via SFTP from `cloud.gov`.
+3. Relevant data is processed and stored in the PostgreSQL database at **8:30 AM EST**.
+4. The system matches reviews with grant recipients and auto-generates goals.
+5. The HSES system processes its copy at **8:00 AM EST** to ensure visibility.
+6. Specialists can manually add objectives based on compliance citations.
+
+## Key System Components
+- **IT-AMS**: The external system that generates monitoring review data.
+- **TTA Hub**: Processes and displays monitoring data for grant recipients.
+- **HSES**: A system that also receives and presents monitoring data.
+- **PostgreSQL Database**: Stores imported monitoring review data.
+- **SFTP**: Secure transfer mechanism for IT-AMS data.
+
+## Database Schema (Simplified)
+| **Table** | **Purpose** |
+|-----------|------------|
+| `MonitoringReviews` | Stores review metadata. |
+| `MonitoringFindings` | Stores findings per review. |
+| `MonitoringReviewLinks` | Links reviews with other entities. |
+| `MonitoringReviewStatusLinks` | Tracks status changes for reviews. |
+
+## Import Process
+- **Schedule:** 8:30 AM EST.
```

Comment:
I think this runs at 10:30pm EST unless I'm reading the cron schedule wrong here:

https://github.com/HHS/Head-Start-TTADP/blob/main/src/lib/maintenance/common.js#L567-L615

Reply:
for reference, where this is actually used to schedule the import download job:

https://github.com/HHS/Head-Start-TTADP/blob/main/src/lib/maintenance/import.ts#L226-L267